Makefile

```diff
@@ -0,0 +1,45 @@
+GO ?= $(shell command -v go 2> /dev/null)
+DEP ?= $(shell command -v dep 2> /dev/null)
+
+PACKAGES=$(shell go list ./...)
+
+## Checks the code style, tests, builds and bundles the plugin.
+all: check-style
+
```

Comment:
There's an extra blank line here. Maybe we need a lint for Makefiles 😛

Reply:
done